### PR TITLE
Migrate Redshift virtual schema to its own repository

### LINKED ISSUE #1: Migrate Redshift dialect to its own repo
Migrate Redshift dialect to its own repo

### PR DESCRIPTION
**Pull request by chiaradiamarcelo**
Closes #1.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+language: java
+
+# Setting sudo to false will cause Travis to use Containers.
+# To use Docker's privileged mode, we need to enable sudo.
+sudo: required
```
Comment (AnastasiiaSergienko) [inline .travis.yml]: I think we don't need this because we don't have integration tests

File: README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Aredshift-virtual-schema&metric=coverage)](https://sonarcloud.io/dashboard?id=com.exasol%3Aredshift-virtual-schema)
+[![Duplicated Lines (%)](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Aredshift-virtual-schema&metric=duplicated_lines_density)](https://sonarcloud.io/dashboard?id=com.exasol%3Aredshift-virtual-schema)
+[![Lines of Code](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Aredshift-virtual-schema&metric=ncloc)](https://sonarcloud.io/dashboard?id=com.exasol%3Aredshift-virtual-schema)
+
+# Overview
+
+The **Redshift Virtual Schema** provides an abstraction layer that makes an external [AWS Redshift](https://aws.amazon.com/redshift/) data source accessible from an Exasol database through regular SQL commands. The contents of the external AWS Redshitf data source are mapped to virtual tables which look like and can be queried as any regular Exasol table.
```
Comment (AnastasiiaSergienko) [inline README.md]:
```suggestion
The **Redshift Virtual Schema** provides an abstraction layer that makes an external [AWS Redshift](https://aws.amazon.com/redshift/) data source accessible from an Exasol database through regular SQL commands. The contents of the external AWS Redshift data source are mapped to virtual tables which look like and can be queried as any regular Exasol table.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+If you want to set up a Virtual Schema for a different database system, please head over to the [Virtual Schemas Repository][virtual-schemas].
+
+## Features
+
+* Access a AWS Redshitf data source in read only mode from an Exasol database, using a Virtual Schema.
```
Comment (AnastasiiaSergienko) [inline README.md]:
```suggestion
* Access an AWS Redshift data source in read-only mode from an Exasol database, using a Virtual Schema.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+| [Exasol Error Reporting][exasol-error-reporting]                   | Creating unified error messages                        | MIT License                                      |
+
+### Test Dependencies
+
+| Dependency                                                         | Purpose                                                | License                       |
+|--------------------------------------------------------------------|--------------------------------------------------------|-------------------------------|
+| [Java Hamcrest](http://hamcrest.org/JavaHamcrest/)                 | Checking for conditions in code via matchers           | BSD License                   |
+| [JUnit](https://junit.org/junit5)                                  | Unit testing framework                                 | Eclipse Public License 1.0    |
+| [Mockito](http://site.mockito.org/)                                | Mocking framework                                      | MIT License                   |
+| [Exasol Hamcrest Result Set Matcher][exasol-hamcrest]              | Hamcrest result set matcher for testing                | MIT License                   |
```
Comment (AnastasiiaSergienko) [inline README.md]: Is it used here? I think this is for integration tests

File: doc/changes/changes_1.0.0.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# RedShift Virtual Schema 1.0.0, released 2021-01-15
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_1.0.0.md]:
```suggestion
# Redshift Virtual Schema 1.0.0, released 2021-01-15
```

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.exasol</groupId>
+	<artifactId>redshift-virtual-schema</artifactId>
+	<version>1.0.0</version>
+	<name>Virtual Schema for RedShift</name>
```
Comment (AnastasiiaSergienko) [inline pom.xml]:
```suggestion
	<name>Virtual Schema for Redshift</name>
```

**Review by AnastasiiaSergienko — CHANGES_REQUESTED** [review]

File: doc/changes/changes_1.0.0.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+# Redshift Virtual Schema 1.0.0, released 2021-01-15
```
Comment (chiaradiamarcelo) [inline doc/changes/changes_1.0.0.md]:
```suggestion
# Redshift Virtual Schema 1.0.0, released 2021-01-18
```

**Review by AnastasiiaSergienko — APPROVED** [review]

**Pull request merged**